cdk/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "test-update": "jest -u",
     "format": "prettier --write \"{lib,bin}/**/*.ts\"",
-    "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
+    "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern --ignore-pattern lib/__snapshots__",
```

Comment:
Without this change, the linter fails with:

```log
  0:0  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/lib/__snapshots__/registration.test.ts.snap` using `parserOptions.project`: <tsconfigRootDir>/../../../../../users/akash_askoolum/code/mobile-n10n/cdk/tsconfig.json
The extension for the file (`.snap`) is non-standard. You should add `parserOptions.extraFileExtensions` to your config
```

I think `eslint` is currently in a strange state as the `package.json` has:

```
devDependencies: {
  "eslint": "^8.16.0"
}
```

However, `yarn dlx eslint --version` returns `v9.39.1`. The `^` shouldn't allow a major version difference!

This is a temporary change to get CI working. It should be removed in a follow up PR that correctly installs `eslint`.